LCM/scripts/PerformInventory.py

```diff
@@ -96,15 +115,14 @@ def usage():
     parameters.append("PerformInventory")
 
 if len(dsc_sysconfdir) < 10:
-    # something has gone horribly wrong. error out before we attempt to remove the entire file system
-    print("Error: Something has gone horribly wrong with the directory paths.")
-    sys.exit(1)
+    # something has gone wrong with the directory paths. error out before we attempt to remove the entire file system
+    exitWithError("Error: Something has gone wrong with the directory paths. Exiting PerformInventory.")
 
-# If inventory lock files already exists update its permissions.  
+# If inventory lock files already exists update its permissions.
 if os.path.isfile(inventorylock_path):
```

Comment:
Let us add some log messages here since we know we have seen issues with file permissions in the past.

Reply:
Done